src/main/java/hudson/plugins/audit_trail/AuditTrailFilter.java

```diff
@@ -122,6 +121,20 @@ private void logRequest(HttpServletRequest request, User user) {
         }
     }
 
+    private boolean isShouldDisplayUserName() {
+        var configuration = getConfiguration();
+        return configuration != null && configuration.shouldDisplayUserName();
+    }
+
+    @CheckForNull
+    private AuditTrailPlugin getConfiguration() {
+        if (Jenkins.get().getInitLevel().compareTo(EXTENSIONS_AUGMENTED) >= 0) {
+            return ExtensionList.lookupSingleton(AuditTrailPlugin.class);
```

Comment:
Naive question (I'm not familiar enough with the Jenkins framework to ascertain that), what's the performance cost of the lookup?

Reply:
Fair question: once initialized it is a map lookup, so fairly quick. To be on the safe side and considering how often this is called, I can rework to initialize the field once it is safe.